cloud/openstack/os_stack.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python
+#coding: utf-8 -*-
+
+# (c) 2016, Mathieu Bultel <mbultel@redhat.com>
+# (c) 2016, Steve Baker <sbaker@redhat.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+from time import sleep
+try:
+    import shade
+    HAS_SHADE = True
+except ImportError:
+    HAS_SHADE = False
+
+DOCUMENTATION = '''
+---
+module: os_stack
+short_description: Add/Remove Heat Stack
+extends_documentation_fragment: openstack
+version_added: "2.1"
+author: "Mathieu Bultel (matbu), Steve Baker (steveb)"
+description:
+   - Add or Remove a Stack to an OpenStack Heat
+options:
+    state:
+      description:
+        - Indicate desired state of the resource
+      choices: ['present', 'absent']
+      required: false
+      default: present
+    name:
+      description:
+        - Name of the stack that should be created
+      required: true
+      default: None
```

Comment:
Also it would be good to specify the limitations on the name here. Can it have spaces?